kong/db/dao/key_sets.lua

```diff
@@ -27,45 +157,72 @@ function key_sets:each(size, options)
   return self.super.each(self, size, options)
 end
 
+function key_sets:start_rotation_timer()
+  kong.log.notice("starting key rotation timer")
+  return timer_every(ROTATION_TIMER, self.rotate_all)
```

Comment:
Should we use named one instead:
https://github.com/Kong/lua-resty-timer-ng#named_every